undertow/src/test/resources/tls/Readme.txt

```diff
@@ -6,4 +6,6 @@ All keystores and entries share the same password 'Elytron'.
 
 ca.truststore - Contains the self signed certificate of the certificate authority.
 scarab.keystore - Contains an alias 'scarab'
-ladybird.keystore - Contains and alias 'ladybird' 
\ No newline at end of file
+ladybird.keystore - Contains and alias 'ladybird'
+
+The keystore 'tiger.keystore' can be used in tests where only beetles are valid.
```

Comment:
The other key stores are based on the certificate authority maintained within the Elytron project so we can cross reference there how the contents are created - for this tiger keystore is there any additional information we can add about how it is created and how to correctly re-create it.